packages/wow-combat-log-parser/src/types.ts

```diff
@@ -3,7 +3,7 @@ import { ArenaMatchStart } from './actions/ArenaMatchStart';
 import { CombatAction } from './actions/CombatAction';
 import { CombatantInfoAction } from './actions/CombatantInfoAction';
 
-export type WowVersion = 'tbc' | 'shadowlands';
+export type WowVersion = 'tbc' | 'dragonflight';
```

Comment:
if the idea is to drop previous expansions, then maybe we should call this enum value "retail"?

Reply:
yeah i think that's probably fine, you can always ref the build # itself inside the log if you really need a version